tests/contest/runtimetest/src/tests.rs

```diff
@@ -546,6 +547,44 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
     }
 }
 
+pub fn validate_process_user(spec: &Spec) {
+    let process = spec.process().as_ref().unwrap();
+
+    let uid = unsafe { getuid() };
+    let gid = unsafe { getgid() };
+    let current_umask = unsafe { libc::umask(0) };
+    unsafe { libc::umask(current_umask) };
+
+    if process.user().uid().ne(&uid) {
```

Comment:
I think you should be able to directly use `!=` operator. Same for below.

Reply:
I fix the code this commit.

https://github.com/youki-dev/youki/commit/df941cd6c3ca9b6cd1a09455c07870b687a83561